packages/amplify_api/lib/src/graphql/helpers/graphql_helpers.dart

```diff
@@ -13,12 +13,4 @@
  * permissions and limitations under the License.
  */
 
-import 'package:amplify_api_plugin_interface/amplify_api_plugin_interface.dart';
-import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
-
-class ModelQueriesFactory extends ModelQueriesInterface {
-  @override
-  GraphQLRequest<T> get<T extends Model>(ModelType<T> modelType, String id) {
-    throw UnimplementedError("get() has not been implemented.");
-  }
-}
+export 'model_queries.dart';
```

Comment:
Is this file just to include things like "model_mutations" and "model_subscriptions" later? If so, can those things be directly exported from amplify_api.dart and reduce the need for this file?

Reply:
Yeah, that was the idea. We can forego it.